packages/labs/signals/README.md

```diff
@@ -0,0 +1,208 @@
+# @lit-labs/signals
+
+TC39 Signals Proposal integration for Lit.
+
+> **Warning** `@lit-labs/signals` is part of the Lit Labs set of packages – it is published
+> in order to get feedback on the design and may receive breaking changes.
+>
+> RFC: https://github.com/lit/rfcs/blob/main/rfcs/0005-standard-signals.md
+>
+> Give feedback: https://github.com/lit/lit/discussions/NNNN
+
+## The TC39 Signals Proposal
+
+The [TC39 Signals Proposal](https://github.com/tc39/proposal-signals) is a
+proposal to add standard signals to the JavaScript language.
+
+This is very exciting for web components, since it means that different web
+components that don't use the same libraries can interoperably watch and produce
+signals.
+
+It also means that many existing state management systems and observability
+libraries that might currently each require their own adapter to integrated with
+the Lit lifecycle, might converge on using standard signals so that we only need
+one Lit adapter, and eventually no adapter at all as support for signals is
+directly added to Lit
```

Comment:
I should probably mention that this is really integration with the signals _polyfill_, and that this package depends on it. Might want to mention that there really needs to be only one copy of the polyfill in an npm installation, otherwise signal tracking breaks.

Reply:
Yeah, I think we need to more clearly delineate "this is what we hope standard signals will bring" vs "this is what you get if you use @lit-labs/signals today"